RiotNSE/BuildSettings.swift

```diff
@@ -0,0 +1,355 @@
+// 
```

Comment:
I don't understand why you added this BuildSettings in RiotNSE?
What is the purpose?

Reply:
ok, we need this occurence to handle correctly this config in the RiotNSE target which is now shared between Tchap and Btchap